openbb_sdk/openbb/package/futures.py

```diff
@@ -87,107 +86,3 @@ def curve(
             "/futures/curve",
             **inputs,
         )
-
```

Comment:
Is it supposed to drop this method?

Reply:
Nah. A missing letter on the init caused it. Fixing it.